sphinx_book_theme/__init__.py

```diff
@@ -250,9 +250,14 @@ def add_header_level_recursive(ul, level):
         else:
             toc_out = soup.prettify()
 
+        if not context["theme_toc_title"]:
+            raise ValueError(
+                "'toc_title' key cannot be empty. Please set a non-empty value."
+            )
+
         out = f"""
         <div class="tocsection onthispage pt-5 pb-3">
-            <i class="fas fa-list"></i> { context["translate"]('Contents') }
+            <i class="fas fa-list"></i> { context["theme_toc_title"] }
```

Comment:
Could we keep the `translate` bit? I think that this would attempt to auto-translate the title for versions of the book that are in different languages. Is that right? Is there any downside to keeping the `translate`?

Reply:
You are right, `translate` is a function with signature `<function get_translation.<locals>.gettext`. So, it should auto-translate.

Reply:
Have re-added it https://github.com/executablebooks/sphinx-book-theme/pull/299/files#diff-1463927e2c28d59a6cedd041803bdfd110650e761a5eda7b5db2b2d946b82949R261